src/client/java/teammates/client/scripts/DataMigrationForFeedbackQuestionsToQuestions.java

```diff
@@ -0,0 +1,137 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import javax.jdo.Query;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Assumption;
+import teammates.logic.api.Logic;
+import teammates.storage.api.FeedbackQuestionsDb;
+import teammates.storage.api.QuestionsDb;
+import teammates.storage.datastore.Datastore;
+import teammates.storage.entity.FeedbackQuestion;
+
+/**
+ * Script to create a Question copy of old FeedbackQuestions.
+ * 
+ */
+public class DataMigrationForFeedbackQuestionsToQuestions extends RemoteApiClient {
+    
+    private final boolean isPreview = true;
+    
+    private enum ScriptTarget {
+        BY_TIME, BY_COURSE, ALL;
+    }
+    
+    ScriptTarget target = ScriptTarget.BY_TIME;
```

Comment:
Maybe set the default target to `ALL`, since we intend to migrate all of the old questions